sky/shell/shell.cc

```diff
@@ -16,7 +16,9 @@
 #include "mojo/message_pump/message_pump_mojo.h"
 #include "sky/shell/ui/engine.h"
 #include "sky/shell/switches.h"
+#if !defined(OS_LINUX)
 #include "ui/gl/gl_surface.h"
```

Comment:
The right fix is to move the call to `gfx::GLSurface::InitializeOneOff` into one of the platform-specific files.  We can land the patch this way if that's too fiddly and I can do it in a followup.